guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -54,7 +54,10 @@ You can use the `no limit` value to show all entitlements.
 ----
 ====
 
-For hosts in non-Simple Content Access (SCA) organizations, you can use a new report template called *Host - Installed Products*.
-The template displays installed product information along with other metrics unrelated to subscriptions that were previously displayed on the *Subscription - Entitlement Report* page.
-The subscription entitlement report no longer lists any hosts that are in SCA organizations.
-It only outputs information for hosts that are in organizations that do not use SCA.
+Host - Installed Products:: Use this template for hosts in Simple Content Access (SCA) organizations.
+It generates a report with installed product information along with other metrics included in *Subscription - Entitlement Report* except information about subscriptions.
+It only outputs information for hosts in organizations that use SCA.
```

Comment:
This is the only line that's not right, the Host - Installed Products report will report against all hosts, regardless of SCA status.